src/main/java/example/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,24 @@
 package example.dto;
 
 import java.math.BigDecimal;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
 import lombok.Getter;
 import lombok.Setter;
 
 @Setter
 @Getter
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```

for String
check other places